### PR TITLE
Add CAM6 suite definition file

### PR DESCRIPTION
**Pull request by nusbaume**
First pass at creating a CAM6 suite definition file.  A few relevant notes and questions:

1.  I wasn't sure if the ZM convective microphysics scheme is part of CAM6, and thus should be included in this file.  I have left it out for now, but can certainly add it in if need be.

2.  I didn't include any timing calls (e.g. "t_startf"), as I wasn't sure if that is managed by the CCPP framework, or by the host model.

3.  I Added schemes which should only matter if WACCM, WACCM-X, CARMA, and/or MOZART (atmospheric chemistry) are used.  It is unclear to me, though, if they should be in the CAM6 suite definition file, or if they should have their own definition files (e.g. suite_waccm.xml).

4.  This suite definition file needs to be updated whenever the use of logical variables becomes allowed, as many schemes depend on certain namelist settings, while other schemes are embedded inside select-case statements with multiple different possibilities.

Finally, I should also note that I haven't done any sort of testing on this file.  If there are any sort of xmllint or other CCPP tests that I should run before merging just let me know.  Thanks!

### CODE REVIEW

**Comment by nusbaume** [comment]
@gold2718 Whoops!  I forgot to mention that I believe this PR is ready for review again, besides the possible disagreement we have with the subcycle variable names.

(No rush, of course, as I assume you are somewhere over the Atlantic this very moment).

File: suite_cam6.xml [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+        <scheme>tropopause_findChemTrop</scheme>      <!-- Calculate tropopause height -->
+        <scheme>clubb_vertical_grid_inv</scheme>      <!-- Change order of vertical levels (should be manageable by registry/meta file) -->
+        <scheme>clubb_calc_ustar</scheme>             <!-- Only matters for single-column mode -->
+        <scheme>setup_grid_heights_api</scheme>       <!-- Re-calculate CLUBB grid heights -->
+        <scheme>setup_parameters_api</scheme>         <!-- Re-calculate grid-dependent CLUBB parameters -->
+        <scheme>zt2zm_convert</scheme>                <!-- Transition variables on thermodynamic levels to momentum levels (manageable by registry/meta file?) -->
+        <scheme>clubb_var_vert_inv</scheme>           <!-- Change order of vertical levels for CLUBB input variables (should be manageable via the registry/meta file) -->
+        <subcycle loop="clubb_nadv">                  <!-- CLUBB sub-step loop (number of iterations should be calculated in the "clubb_dtime_calc" scheme) -->
+          <scheme>stats_begin_timestep_api</scheme>   <!-- Only matters if "l_stats" is True -->
```
Comment (gold2718) [inline suite_cam6.xml]: Should this be a timestep_init routine?

File: suite_cam6.xml [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+      <scheme>modal_aero_wateruptake_diag</scheme>    <!-- Calculate aerosol water uptake diagnostic variables, and output them to history file -->
+      <!-- **************************** -->
+      <!-- Aerosol wet deposition -->
+      <!-- **************************** -->
+      <scheme>modal_aero_calcsize_sub</scheme>        <!-- Calculate  aerosol size distribution parameters -->
+      <scheme>modal_aero_wateruptake_dr</scheme>      <!-- Repeat "Modal Aerosol water uptake" schemes shown above -->
+      <scheme>wetdep_prec_calc</scheme>               <!-- Calculate precipitation amount that impacts wet deposition -->
+      <scheme>coarse_fact_calc</scheme>               <!-- Calculate "f_act_conv" for coarse mode aerosols (note here the comments don't match the variables) -->
+      <subcycle loop="ntot_amode">                    <!-- Loop over aerosol modes -->
+        <subcycle loop="lphase">                      <!-- Loop over aerosol phase types (interstitial vs cloud-borne -->
```
[the remark this answers is not in the document]
Comment (gold2718) [reply suite_cam6.xml]: I think you can leave the variable names as comments.

**Comment by nusbaume** [comment]
Thanks @gold2718 for the feedback!  I have implemented the requested changes, but given that my fork has been separated from the main repo after it went public, I am no longer able to push commits to this particular PR.  Thus I am going to close this PR and re-open it with my new fork.  Apologies in advance for any confusion this causes!

File: suite_cam6.xml (hunk 1 above)
Comment (nusbaume) [reply suite_cam6.xml]: Given that this scheme is run for every CLUBB sub-step (as opposed to the host model time step), I wasn't sure if it was an official "timestep_init" routine, or if it had to be treated as a regular physics scheme?

**Pull request closed without merging**